### DISCUSSION TITLE
Import Zemax DAT file

### DISCUSSION THREAD

**Discussion by pgaur7** [NanoComp/meep#2161] [Q&A]
I have a source ray-file in .DAT format that I use in Zemax OpticStudio. I would like to import the same in MEEP for nano-optic simulations. Is there a way to convert this file to a source in MEEP?

**Comment by stevengj** [comment]
As far as I know this is an undocumented file format specific to Zemax, so it's not something we're likely to decipher.

In fact, according to the [Zemax documentation](https://support.zemax.com/hc/en-us/articles/1500005486681-Zemax-file-extensions), it's not even a single file format — `.DAT` seems to be a catch-all extension used for a variety of data dumps in different context-dependent formats.

**Reply by pgaur7** [reply]
Thank you for your reply! So would it be possible to create a .ZBF file or any other file format in Zemax that has field data and import it to meep to act as a source?
